package.json

```diff
@@ -765,6 +797,16 @@
                     "dark": "resources/dark/s3/create-bucket.svg"
                 }
             },
+            {
+                "command": "aws.s3.deleteBucket",
```

Comment:
Where's the code for these DeleteFile, DeleteBucket, and UploadFileToParent commands?

Reply:
Sorry, hadn't posted it at the time I posted this. The command impls are in #1156

Reply:
This PR results in a broken state when standing alone from other PRs. It adds menu items to explorer nodes, which if selected, raise an error that the command does not exist. 

Is the intent for this PR to follow 1156?

Reply:
No the intent was that #1156 is the last in this unit (passing all checks).

> This PR results in a broken state when standing alone from other PRs.

Yes, this is the downside of splitting commits by concern without making them standalone. Reverting a commit would cause the code to not compile. 

However, reverts are very rare and this I've found it to be much more practical than building 4 standalone commits. It lets you quickly iterate on multiple concerns without constantly switching branches, rebasing, and rebuilding to keep everything tidy. Or worse, submitting PR#1 only to scrap and refactor everything from there in PR#3. That would be a waste of time for the reviewer.